deploy/kustomize/patches/worker-subs-extraction.yaml

```diff
@@ -0,0 +1,194 @@
+- op: add
+  path: /objects/0/spec/deployments/-
+  value:
+    name: clowder-worker-subs-extraction
+    replicas: ${{WORKER_SUBS_EXTRACTION_REPLICAS}}
+    webServices:
+      public:
+        enabled: false
+      private:
+        enabled: false
+    podSpec:
+      image: ${IMAGE}:${IMAGE_TAG}
+      command:
+        - /bin/bash
+        - -c
+        - > # ${APP_HOME} is `/opt/koku/koku` and is defined in the Dockerfile
+          PYTHONPATH=${APP_HOME}
+          celery -A koku worker --without-gossip -E -l $CELERY_LOG_LEVEL -Q $WORKER_QUEUES
+      env:
+        - name: CLOWDER_ENABLED
+          value: ${CLOWDER_ENABLED}
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              key: aws-access-key-id
+              name: koku-aws
+              optional: false
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: aws-secret-access-key
+              name: koku-aws
+              optional: false
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: ${GOOGLE_APPLICATION_CREDENTIALS}
+        - name: APP_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: DEVELOPMENT
+          value: ${DEVELOPMENT}
+        - name: ENABLE_SUBS_DEBUG
+          value: ${ENABLE_SUBS_DEBUG}
+        - name: CELERY_LOG_LEVEL
+          value: ${CELERY_LOG_LEVEL}
+        - name: KOKU_LOG_LEVEL
+          value: ${KOKU_LOG_LEVEL}
+        - name: UNLEASH_LOG_LEVEL
+          value: ${UNLEASH_LOG_LEVEL}
+        - name: PROMETHEUS_MULTIPROC_DIR
+          value: ${PROMETHEUS_DIR}
+        - name: REQUESTED_BUCKET
```

Comment:
These workers would need access to the subs bucket similar to [what we did here](https://github.com/project-koku/koku/pull/4230/files#diff-20dabfb7f9b39f8ad15f9faa13f99e8f0a81a8f146b383294aaeada5ab76c359L53-L54 ) for the ROS bucket

Reply:
SUBS bucket is added. Thanks for pointing it out.

Reply:
I think we just need it [to the object store](https://github.com/project-koku/koku/pull/4230/files#diff-23b38de8fd89583f8a61f3426216dc745c12d84068fd5fff6f259075ab186269R39-R41) and [give a default like we](https://github.com/project-koku/koku/pull/4230/files#diff-23b38de8fd89583f8a61f3426216dc745c12d84068fd5fff6f259075ab186269R224-R227) did for ROS (for ephemeral basically) so its filled in for the clowdapp:
```
- name: REQUESTED_SUBS_BUCKET
   value: '${S3_SUBS_BUCKET_NAME}'
```

Reply:
and we should also add the [kafka topic like we did here for ROS](https://github.com/project-koku/koku/blob/main/deploy/kustomize/base/base.yaml#L37)

Reply:
Thanks again Corey, I've added these changes!